src/core/Akka.Persistence/Fsm/PersistentFSMBase.cs

```diff
@@ -1059,7 +1059,7 @@ public State ForMax(TimeSpan timeout)
             /// <summary>
             /// TBD
             /// </summary>
-            public List<object> Replies
+            public IReadOnlyCollection<object> Replies
```

Comment:
We should prefer `ReadOnly*` and `Immutable` collections in the public API. But this change could break compatibility